gallery/businesscontinuity-insights/businesscontinuity-insights.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
"businesscontinuity-insights" is not a valid workbook type. if you intend to add a new insights type, it would have to go through the workbooks team, it requires a bunch of work like labels and support ids, etc

this shouldn't actually work if you tried to use it, you'd see warnings in the browser F12 console that this is an unknown type and it would be falling back to "workbooks", and then it would never see this content